defpack tools/Defpack Tools.mod.gml

```diff
@@ -4431,6 +4458,7 @@ with instance_create(x, y, melee ? CustomSlash : CustomProjectile){
         on_anim = nothing
         on_projectile = sword_proj
         on_grenade = sword_proj
+		is_melee = true

```

Comment:
Swords are bolt projectiles; they just happen to clear projectiles. I wouldn't consider them a melee attack.